src/librustdoc/html/render/print_item.rs

```diff
@@ -2267,6 +2271,53 @@ fn document_non_exhaustive<'a>(item: &'a clean::Item) -> impl fmt::Display + 'a
     })
 }
 
+fn document_default_header(
+    cx: &mut Context<'_>,
+    it: &clean::Item,
+    variants: &IndexVec<VariantIdx, clean::Item>,
+) -> String {
+    let tcx = cx.tcx();
+    let cache = cx.cache();
+    let v: Vec<&Impl> = cx.shared.all_impls_for_item(it, it.item_id.expect_def_id());
```

Comment:
Minor: Since we know for a fact that `it` is an enum, you could also use `cache.impls.get(…)` directly which might be a bit cleaner or faster but dunno

Reply:
done